compiler/ballerina-lang/src/main/java/io/ballerina/projects/JBallerinaBackend.java

```diff
@@ -78,6 +78,7 @@
 import static io.ballerina.projects.util.ProjectConstants.BIN_DIR_NAME;
 import static io.ballerina.projects.util.ProjectConstants.DOT;
 import static io.ballerina.projects.util.ProjectUtils.getThinJarFileName;
+import static org.wso2.ballerinalang.compiler.bir.codegen.JvmConstants.CLASS_FILE_SUFFIX;
```

Comment:
I think it is better to create this constant here because if we see the usages it is used in codegen.
<img width="1009" alt="image" src="https://github.com/ballerina-platform/ballerina-lang/assets/33286289/eb933cc6-ebd5-4012-b146-64e6a7f3e1ec">
and project constants says they are constants related to the project directory.
<img width="540" alt="image" src="https://github.com/ballerina-platform/ballerina-lang/assets/33286289/4f434521-a685-42ad-ac88-91e49e53a81f">
Shall we keep it here?

Reply:
+1